mobile/lib/features/welcome/onboarding.dart

```diff
@@ -10,59 +11,60 @@ final themeMode = ValueNotifier(2);
 class CarouselItem {
   final String title;
   final String description;
-  final String imagePath;
-  final double aspectRatio;
+  final Widget imageWidget;
 
-  CarouselItem(this.title, this.description, this.imagePath, this.aspectRatio);
+  CarouselItem(
+    this.title,
+    this.description,
+    this.imageWidget,
+  );
 }
 
 final List<CarouselItem> carouselItems = [
-  CarouselItem("Your keys, your control", "Stay in control of your funds at all time.",
-      "assets/carousel_1.jpg", 30 / 35),
-  CarouselItem("Bitcoin only & Lightning fast.",
-      "The highest level of security, at lightning speed.", "assets/carousel_2.jpg", 30 / 38),
   CarouselItem(
-      "Perpetual futures trading.",
-      "Experience P2P leveraged trading with no counterparty risk.",
-      "assets/carousel_3.jpg",
-      30 / 35),
+      "Welcome Satoshi,",
+      "In 10101, every trade is collateralized and fully self-custodial. \n We use DLCs (Discreet Log Contracts) for this. Your first trade will open a DLC Channel which defines the amount which will be available to trade. ",
+      Image.asset("assets/carousel_1.png")),
+  CarouselItem(
+      "Self-custodial",
+      "The max amount you put at risk will depend on how much you lock up. At the same time, the amount you can win will depend on the amount your channel counterparty will lock up.",
+      const PnlLineChart()),
   CarouselItem(
-      "Hedging and synthetics",
-      "You can now send, receive and hold USDP natively on Lightning.",
-      "assets/carousel_4.jpg",
-      30 / 28),
+      "Welcome to the future",
+      "Once you have an open channel, you can trade with very small amounts. \n In the future you will be able to extend or reduce the channel size (splice-in/splice-out). ",
```

Comment:
It sounds like we _want_ them to trade with very small amounts haha.

I understand that we want to explain all this stuff about the current channel-opening quirks, but this feels out of place in the carousel. A tooltip might be good enough for now. Or simply tackling the FAQ update or the on-boarding flow ASAP.